common/src/main/java/gov/cms/ab2d/common/dto/UserDTO.java

```diff
@@ -0,0 +1,30 @@
+package gov.cms.ab2d.common.dto;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+import java.util.Set;
+
+@Data
+public class UserDTO {
+
+    @NotNull
+    private String username;
+
+    @NotNull
+    private String firstName;
+
+    @NotNull
+    private String lastName;
+
+    @NotNull
+    private String email;
+
+    @NotNull
+    private Boolean enabled;
+
+    @NotNull
+    private Long sponsorId;
+
+    private Set<RoleDTO> roles;
```

Comment:
We won't have users with more than one role. Should we change this to just `String role`?

Reply:
Is it that only users created from the API won't have more than one role or all users in general? The User model now has users - roles as a many to many

Reply:
All users. Many-to-many covers possible future changes